### DISCUSSION TITLE
[PSD12] Exercise 3.5

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#178] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210064979-174c65dd-ae3b-429d-bdca-343ace23e8f6.png)

![image](https://user-images.githubusercontent.com/69389461/210064921-add784a0-b082-4023-8e84-deb790c4f7ce.png)

**Comment by Elia-Belli** [comment]
![210065128-22f0382d-d0ed-4d83-90d1-84cde2b99a7b](https://user-images.githubusercontent.com/107944476/211635115-ba45b6af-9248-4faf-a94d-d07cbf520f3b.png)
Un D-flipflop campiona il segnale D solamente sul fronte (generalmente positivo) del clock

**Comment by fraxb04** [comment]
corretto
